src/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild/ProjectReferenceInformationProvider.cs

```diff
@@ -62,9 +60,22 @@ public class ProjectReferenceInformationProvider
             return projectReferenceInformation;
         }
 
+        private static string EnsurePathRooted(string path, string basePath)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                return path;
```

Comment:
throw error. If the relative path is null or empty, we don't know which csproj to use.